tests/zfs-tests/tests/functional/delegate/delegate_common.kshlib

```diff
@@ -715,10 +715,15 @@ function verify_fs_snapshot
 	fi
 	log_must zfs destroy $snap
 
-	typeset snapdir=${mntpt}/.zfs/snapshot/snap.$stamp
-	user_run $user mkdir $snapdir
-	if ! datasetexists $snap ; then
-		return 1
+	if [[ $is_linux == 1 ]] && \
+	    [[ $(</sys/module/zfs/parameters/zfs_admin_snapshot) == 0 ]]; then
+		log_note zfs_admin_snapshot is disabled.
```

Comment:
Rather than always skipping these tests, let's instead enable this functionality to make sure run the tests.  Something simple like this for the test group should be fine.

in `setup.sh`
```
log_must set_tunable64 zfs_admin_snapshot 1
```

in `cleanup.sh`
```
log_must set_tunable64 zfs_admin_snapshot 0
```